group-24_Palpitate_app_0.py

```
@@ -0,0 +1,64 @@
+from flask import Flask, request, Response, abort, send_file, jsonify, render_template, url_for, redirect, send_from_directory
+from werkzeug import secure_filename
+import os, subprocess, re, sys, difflib
+
+app = Flask(__name__)
+
+video_file = sys.argv[1]
+
+@app.route('/')
+def index():
+    return render_template('homepage.html');
```

Ruff (0.13.1)

Statement ends with an unnecessary semicolon

(E703)